pkg/acsengine/defaults.go

```diff
@@ -460,6 +462,12 @@ func setOrchestratorDefaults(cs *api.ContainerService) {
 			}
 		}
 
+		if helpers.IsTrueBoolPointer(o.KubernetesConfig.EnableDataEncryptionAtRest) {
+			if "" == a.OrchestratorProfile.KubernetesConfig.EtcdEncryptionKey {
+				a.OrchestratorProfile.KubernetesConfig.EtcdEncryptionKey = generateEtcdEncryptionKey()
+			}
```

Comment:
If `EtcdEncryptionKey` is provided, is there verification for if the value is base64 encoded?